docs/intro/tutorial01.txt

```diff
@@ -14,7 +14,8 @@ It'll consist of two parts:
 
 We'll assume you have :doc:`Django installed </intro/install>` already. You can
 tell Django is installed and which version by running the following command
-in a shell prompt (indicated by the $ prefix):
```

Comment:
`$` in this refers to the `$` in the below example. I'd change this.

Reply:
It's only an explanation of what is `$` in the above example.

Reply:
The problem is that there is no $ in windows example.

Reply:
OK, let's reopen to allow a longer review.